aws_docker_creds.sh

```diff
@@ -13,6 +13,25 @@ cat << EOF > ~/.aws/config
 region = $AWS_REGION
 EOF
 
+# For multi account aws setups, use primary credentials to assume the role in
+# the target account
+if [[ -n $AWS_STS_ROLE || -n $AWS_STS_ACCOUNT ]]; then
```

Comment:
The other AWS variables are still required to call "aws sts" so I figured it made more sense to check for these optional ones after the other requirements were met